dashboard/app/controllers/courses_controller.rb

```diff
@@ -98,7 +98,8 @@ def update
       unit_group.student_resources = params[:studentResourceIds].map {|id| Resource.find(id)} if params.key?(:studentResourceIds)
     end
     # Convert checkbox values from a string ("on") to a boolean.
-    [:has_verified_resources, :has_numbered_units, :visible, :is_stable].each {|key| params[key] = !!params[key]}
+    [:has_verified_resources, :has_numbered_units].each {|key| params[key] = !!params[key]}
+    [:visible, :is_stable].each {|key| params[key] = params[key]}
```

Comment:
What does this line do?

Reply:
Lol- nothing now. Before I switched over to the new course editing I had to `to_bool` here